assets/prototype/domain/eventEditor/data/queries/datetimes/test/useDatetimes.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect } from 'react';
+import { useApolloClient } from '@apollo/react-hooks';
+import { renderHook } from '@testing-library/react-hooks';
+
+import useDatetimes from '../useDatetimes';
+import contextWrapper from './contextWrapper';
+import { successMocks, setup, cleanup, nodes, request, edge } from './data';
+import { useStatus, TypeName } from '../../../../../../application/services/apollo/status';
```

Comment:
Not directly related to this PR, but we need to do something with these `../../../../../../` :smiley:

Reply:
We can convert those hooks to a package.

Reply:
Yes, good suggestion.
We can do monorepo as in GB or as in this example: https://github.com/jannikbuschke/lerna-react-typescript-sample